build.gradle

```diff
@@ -19,7 +19,7 @@ buildscript {
         compileSdkVersion = 30
         targetSdkVersion = 30
 
-        activityVersion = '1.3.0-alpha02'
+        activityVersion = '1.2.0-rc01'
```

Comment:
The revert here wasn't what I had asked for - I asked for `1.2.0`. We still want to upgrade from the RC to stable release, there's just no reason to go up to Activity 1.3

Reply:
Ok. Sorry